nodes/FormX/actions/document/commonProperties.ts

```diff
@@ -4,11 +4,13 @@ export const commonProperties = (
 	extraAdditionalFields: INodeProperties[] = [],
 ): INodeProperties[] => [
 	{
-		displayName: 'Image URL',
-		name: 'imageUrl',
+		displayName: 'Binary Data Field',
+		name: 'binaryDataField',
 		type: 'string',
-		default: '',
-		placeholder: 'https://formextractorai.com/sample-invoice-1.d551279a.jpg',
+		default: 'data',
```

Comment:
I think we should support both image URL or binary file...? Or does n8n support uploading the file directly?

Reply:
Yes, both are supported. But by default binary file is expected.

If want to use image url, can pass the url in additional field

Reply:
Gotcha